website/src/lib/transformation.ts

```diff
@@ -208,42 +207,3 @@ export function transformWithMapping(
   // Recursively walk the mapping until all nested transformations are applied
   return transformNode(mapping, depth) as Record<string, JsonValue>;
 }
-
-/**
- * Maps JSON data from one form schema to another using the common data format as an intermediary.
- *
- * @param data - The source form data
- * @param sourceId - The ID of the source form schema
- * @param targetId - The ID of the target form schema
- * @returns A TransformOutput object containing the transformation results
- */
-export function mapJson(
```

Comment:
Moves this to `playground/index.tsx` since it depends on the Vite glob function which only runs on build. This allows us to use the `transformWithMapping()` function in this file in the local script `npm run validate:schemas`